cypress/e2e/basic-starter.cy.ts

```diff
@@ -132,7 +132,7 @@ describe('Taxonomy term page', () => {
   it('should render places for Office (place type)', () => {
     cy.visit('/place_type/office');
     cy.get('h1').should('contain.text', 'Office');
-    cy.get('h2', { timeout: 5000 }).should('be.visible');
+    cy.get('h2', { timeout: 8000 }).should('be.visible');
```

Comment:
This test failed on one of the builds on the first test run so that's why I increased the wait for now